src/substrate/events/service-request/commands/processed-service-request/processed-service-request.handler.ts

```diff
@@ -44,11 +44,12 @@ export class ProcessedServiceRequestHandler
 
     const service_request = body.hits?.hits[0]?._source || null;
 
-    if (service_request != null) {
+    if (service_request !== null) {
```

Comment:
too many codes inside this `if` statement, please make it simpler by using the return instead